src/main/java/net/pms/database/MediaTableTVSeries.java

```diff
@@ -424,6 +311,190 @@ public static Long set(final Connection connection, final JsonObject tvSeries, f
 		return null;
 	}
 
+	/**
+	 * Updates an existing row with information from our API.
+	 *
+	 * @param connection the db connection
+	 * @param serieMetadata
+	 * @param tvSeriesId
+	 */
+	public static void updateAPIMetadata(final Connection connection, final TvSerieMetadata serieMetadata, final Long tvSeriesId) {
```

Comment:
Should be `series` not `serie` (the s does not mean plural in this case, because English language is silly...)